### DISCUSSION TITLE
A goal to monitor traffic from an external page

### DISCUSSION THREAD

**Discussion by bzoum** [plausible/analytics#975] [Anything Else]
Hello,
Is it possible to setup a goal to monitor the traffics from a specific external page to a page of our website (to check if a backlink brings traffic for example)?

**Comment by metmarkosaric** [comment]
UTM tags are best for that kind of stuff. See https://plausible.io/docs/manual-link-tagging

**Reply by bzoum** [reply]
Thank you but it does not work because I don't control how the link is written on the external site.
The use case: my website is showcase on another website. I'm happy, but I would like to know what traffics it will generate.

**Reply by metmarkosaric** [reply]
ahhh ok in that case the best you can do is check the Top Sources report and see if you can find that website there as a referral. then click on it to segment your stats by traffic from that website only
